src/staging/src/components/park/advisory.js

```diff
@@ -56,6 +56,7 @@ export default function Advisory({ advisories }) {
             src={alertIcon}
             aria-label="park access status"
             className="park-overview-icon"
+            alt="icon"
```

Comment:
Decorative images usually don’t need alt text.
I believe `alt=""` is actually better for screen readers.

Reply:
Thanks for catching this. This is a MUI component, and if you put `alt=""` in it strips it to `alt`, which is why I went with the above. I later discovered that from a screen-reader and SEO POV `alt=""` and `alt` are equivalent. So I'll switch all of those.